addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -113,7 +113,7 @@ private _id = switch (_type) do {
     };
     case "featurecamera": {
         if (_applyRetroactively) then {
-            [GVAR(oldUnit), call CBA_fnc_getActiveFeatureCamera] call _function;
+            [GVAR(oldUnit), call CBA_fnc_getActiveFeatureCamera, ""] call _function;
```

Comment:
Feature camera only reports the new cam, not the old state, though I suppose it could do that. Then you need to change below as well.